pkg/resolution/resource/crd_resource.go

```diff
@@ -54,7 +55,10 @@ var _ Requester = &CRDRequester{}
 func (r *CRDRequester) Submit(ctx context.Context, resolver ResolverName, req Request) (ResolvedResource, error) {
 	rr, _ := r.lister.ResolutionRequests(req.Namespace()).Get(req.Name())
 	if rr == nil {
-		if err := r.createResolutionRequest(ctx, resolver, req); err != nil {
+		if err := r.createResolutionRequest(ctx, resolver, req); err != nil &&
+			// If the request already exists then we can assume that is in progress.
```

Comment:
can we add the reason why it may already exist in the comment? I personally didn't get the idea until reading the commit message 😄

Reply:
Ok, I add a simple description.